source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -185,8 +187,8 @@ export const handleHardwareWalletRequests = async (
         if (event.payload && event.payload.device) {
           TrezorConnect.uiResponse({
             type: UI.RECEIVE_PASSPHRASE,
-            // @ts-ignore ts-migrate(2322) FIXME: Type '{ value: string; passphraseOnDevice: true; }... Remove this comment to see the full error message
             payload: {
+              save: true,
```

Comment:
This has become a required parameter now, however whether it's set to `true` or `false` does not make any difference. The issue was the same on 8.2.4